pkg/fio/fio.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"os"
+	"k8s.io/apimachinery/pkg/labels"
```

Comment:
[nit] this should probably move to the list below with other third-party imports

Not blocking the PR, we can fix it as a follow-up